res/values/strings.xml

```diff
@@ -161,119 +161,119 @@
     <string name="mute_for_seven_days">Mute for 7 days</string>
     <string name="mute_forever">Mute forever</string>
 
-    <string name="share_location_once">once</string>
-    <string name="share_location_for_5_minutes">for 5 minutes</string>
-    <string name="share_location_for_30_minutes">for 30 minutes</string>
-    <string name="share_location_for_one_hour">for 1 hour</string>
-    <string name="share_location_for_two_hours">for 2 hours</string>
-    <string name="share_location_for_six_hours">for 6 hours</string>
+    <string name="share_location_once">Once</string>
+    <string name="share_location_for_5_minutes">For 5 minutes</string>
+    <string name="share_location_for_30_minutes">For 30 minutes</string>
+    <string name="share_location_for_one_hour">For 1 hour</string>
+    <string name="share_location_for_two_hours">For 2 hours</string>
+    <string name="share_location_for_six_hours">For 6 hours</string>
 
     <plurals name="ask_send_following_n_files_to">
         <item quantity="one">Send the following file to %s?</item>
         <item quantity="other">Send the following %d files to %s?</item>
     </plurals>
     <string name="file_saved_to">File saved to \"%1$s\".</string>
 
-    <string name="videochat">Video chat</string>
+    <string name="videochat">Video Chat</string>
     <string name="videochat_invite_user_to_videochat">Invite %1$s to a video chat?</string>
     <string name="videochat_invite_user_hint">This requires a compatible app or a compatible browser on both ends.</string>
     <string name="videochat_contact_invited_hint">%1$s invited to a video chat.</string>
     <string name="videochat_you_invited_hint">You invited to a video chat.</string>
     <string name="videochat_will_open_in_your_browser">This video chat will open in your browser.</string>
-    <string name="videochat_tap_to_join">Tap to join</string>
-    <string name="videochat_tap_to_open">Tap to open</string>
-    <string name="videochat_instance">Video chat instance</string>
-    <string name="videochat_instance_placeholder">Your video chat instance</string>
-    <string name="videochat_instance_explain">If a video chat instance is defined, you can start a video chat from each chat. Video chats require a compatible app or a compatible browser on both ends.\n\nExamples: https://meet.jit.si/$ROOM or basicwebrtc:https://your-server</string>
+    <string name="videochat_tap_to_join">Tap to Join</string>
+    <string name="videochat_tap_to_open">Tap to Open</string>
+    <string name="videochat_instance">Video Chat Instance</string>
+    <string name="videochat_instance_placeholder">Your Video Chat Instance</string>
+    <string name="videochat_instance_explain">If a video chat Instance is defined, you can start a video chat from each chat. video chats require a compatible app or a compatible browser on both ends.\n\nExamples: https://meet.jit.si/$ROOM or basicwebrtc:https://your-server</string>
```

Comment:
"video" should probably stay uppercased as it's start-of-sentence?

And "Instance" should probably stay lowercased as it's mid-sentence?

Reply:
that's a typo that sneaked in, everywhere else we say "video chat instance" when inside a sentence.